backend/app/models/budget.rb

```diff
@@ -0,0 +1,103 @@
+# budgets table
+class Budget < ActiveRecord::Base
+
+  validates :name,        presence: true
+  validates :description, presence: true
+  validates :contact,     presence: true
+  validates :email,       presence: true
+  validates :phone,       presence: true
+
+  # Return all budgets.
+  #
+  # @return all budgets
+  def self.find_all
+    Budget.order(:name)
+  end
+
+  # Return all budgets beginning with fitst letter l ('*' as non-alphanumeric wildcard).
```

Comment:
misspelled "first"

Reply:
Done.

Reply:
I'm not seeing the change. Did you check this in?

Reply:
My "done." comments should be "done on dev."  I was waiting to clear out the other comments on the PR to check in everything at once.

About to check in now.

Reply:
better to check it in so that I can resolve it